src/bridgePausing.ts

```diff
@@ -1,24 +1,50 @@
 import axios from 'axios';
 import {Command, Option} from 'commander';
 import {ConfigUrl} from "./constants";
-import {RawConfig} from "@buildwithsygma/sygma-sdk-core";
+import {getWalletsForDifferentProviders, deriveWalletsFromMnemonic, sendPauseTransactions} from "./utils";
 
 const program = new Command();
-console.log("pero");
 
 program
   .name("pause-bridge")
   .description("Pauses all bridge instances across all networks")
-  .argument("<string>", "mnemonic or private key of the wallet")
+  .version("0.0.1")
+
+program
+  .command("pause")
   .addOption(
-    new Option('--environment, -e', 'Environment on which to pause bridge instances')
+    new Option('-e, --environment <environment>', 'Environment on which to pause bridge instances')
       .choices(['devnet', 'testnet', 'mainnet'])
   )
-  .action(async (mnemonic: string, environment: keyof typeof ConfigUrl) => {
+  .addOption(
+    new Option('-pk, --private-key <privateKey>', 'Private key to use for signing transactions')
+  )
+  .addOption(
+    new Option('-m, --mnemonic <mnemonic>', 'Mnemonic to use for signing transactions').conflicts('private-key')
+  )
+  .action(async (configs: any) => {
     try {
-      console.log("pero");
-      const response = await axios.get(ConfigUrl[environment]) as unknown as RawConfig;
-      console.log(response)
+      const network: keyof typeof ConfigUrl = configs.environment;
+      const {
+        privateKey,
+        mnemonic
+      } = configs;
+
+      const response = await axios.get(ConfigUrl[network]) as any;
+      const networks = response.data.domains.filter((network: any) => network.type === "evm"); // just evms for now
```

Comment:
Same for network, I think there is a type `Domain` in sygma SDK, import it from there